buildsettings.yaml

```diff
@@ -1,8 +1,8 @@
 version:
   major: 1
-  minor: 1
-  patch: 1
-  prerelease: "snapshot"
+  minor: 2
+  patch: 0
+  prerelease: "rc1"
```

Comment:
This is intentionally a release candidate, right?

Reply:
right now, yeah, will switch it to `release` before merge